Marlin/src/lcd/extensible_ui/ui_api.h

```diff
@@ -49,161 +49,211 @@ typedef const __FlashStringHelper *progmem_str;
+    void      setAxisMaxJerk_mm_s            (const float, const axis_t);
+    void      setAxisMaxJerk_mm_s            (const float, const extruder_t);
   #endif
 
-  void setActiveTool(uint8_t extruder, bool no_move);
-  uint8_t getActiveTool();
+  extruder_t  getActiveTool                  ();
+  void        setActiveTool                  (const extruder_t, bool no_move);
+
 
   #if HOTENDS > 1
-    float getNozzleOffset_mm(const axis_t axis, uint8_t extruder);
-    void setNozzleOffset_mm(const axis_t axis, uint8_t extruder, float offset);
+    float     getNozzleOffset_mm             (             const axis_t, const extruder_t);
+    void      setNozzleOffset_mm             (const float, const axis_t, const extruder_t);
   #endif
 
   #if ENABLED(BABYSTEP_ZPROBE_OFFSET)
-    float getZOffset_mm();
-    void setZOffset_mm(const float zoffset_mm);
-    void incrementZOffset_steps(const int16_t babystep_increment);
+    float     getZOffset_mm                  ();
+    void      setZOffset_mm                  (const float);
+    void      addZOffset_steps               (const int16_t);
   #endif
 
   #if ENABLED(BACKLASH_GCODE)
-    float getAxisBacklash_mm(const axis_t axis);
-    void setAxisBacklash_mm(const axis_t axis, float distance);
+    float     getAxisBacklash_mm             (             const axis_t);
+    void      setAxisBacklash_mm             (const float, const axis_t);
 
-    float getBacklashCorrection_percent();
-    void setBacklashCorrection_percent(float percent);
+    float     getBacklashCorrection_percent  ();
+    void      setBacklashCorrection_percent  (const float);
 
     #ifdef BACKLASH_SMOOTHING_MM
-      float getBacklashSmoothing_mm();
-      void setBacklashSmoothing_mm(float distance);
+      float   getBacklashSmoothing_mm        ();
+      void    setBacklashSmoothing_mm        (const float);
     #endif
   #endif
 
   #if ENABLED(FILAMENT_RUNOUT_SENSOR)
-    bool isFilamentRunoutEnabled();
-    void toggleFilamentRunout(const bool state);
+    bool      getFilamentRunoutEnabled       ();
+    void      setFilamentRunoutEnabled       (const bool);
 
     #if FILAMENT_RUNOUT_DISTANCE_MM > 0
-      float getFilamentRunoutDistance_mm();
-      void setFilamentRunoutDistance_mm(const float distance);
+      float   getFilamentRunoutDistance_mm   ();
+      void    setFilamentRunoutDistance_mm   (const float);
     #endif
   #endif
 
-  // This safe_millis is safe to use even when printer is killed (as long as called at least every 1 second)
-  uint32_t safe_millis();
-  void delay_us(unsigned long us);
-  void delay_ms(unsigned long ms);
-  void yield(); // Within lengthy loop, call this periodically
+  /* Delay and timing routines */
+  /* Should be used by the EXTENSIBLE_UI to safely pause or measure time */
+  /* safe_millis must be called at least every 1 sec to guarantee time */
+  /* yield should be called within lengthy loops */
```

Comment:
C-style per-line comments aren't necessary for Marlin code. Use C++ single line or use a single block style comment when the length over 2-3 lines.

Reply:
Yeah. Not sure why I did it that way. Normally I use single block style comment. I generally like to keep the asterisks running all the way down the left, however since I feel it helps make the comment visually stand out.